projects/UWMadison_Ericksen.yaml

```diff
@@ -0,0 +1,5 @@
+Department: Small Molecule Screening Facility
+Description: http://hts.wisc.edu/
```

Comment:
Would it be possible to include some prose here describing the project in addition to the URL?